contract/wizards/contract_manually_create_invoice.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2019 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models, _
+
+
+class ContractManuallyCreateInvoice(models.TransientModel):
+
+    _name = 'contract.manually.create.invoice'
+    _description = 'Contract Manually Create Invoice Wizard'
+
+    invoice_date = fields.Date(string="Invoice Date", required=True)
+    contract_to_invoice_count = fields.Integer(
+        compute="_compute_contract_to_invoice_ids"
+    )
+    contract_to_invoice_ids = fields.Many2many(
+        comodel_name="contract.contract",
+        compute="_compute_contract_to_invoice_ids",
+    )
+
+    @api.depends('invoice_date')
+    def _compute_contract_to_invoice_ids(self):
+        if not self.invoice_date:
+            contract_to_invoice_domain = [('id', '=', False)]
```

Comment:
Perhaps add a comment here (trick to show no invoice when no date has been entered yet)